cacti/tests/test_cacti.py

```diff
@@ -139,3 +132,45 @@ def test_check(aggregator, check):
     aggregator.assert_metric('cacti.rrd.count', value=5, tags=CUSTOM_TAGS)
     aggregator.assert_metric('cacti.hosts.count', value=1, tags=CUSTOM_TAGS)
     aggregator.assert_all_metrics_covered()
+
+
+def test_whitelist(aggregator, check):
```

Comment:
This added test is not related but coverage was below target and whitelist feature didn't have any tests